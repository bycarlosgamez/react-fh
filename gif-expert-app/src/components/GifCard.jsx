import getGifs from '../helper/getGifs';

const GifCard = ({ category }) => {
  getGifs(category);

  return (
    <>
      <h3>{category.toUpperCase()}</h3>
      <p>Description</p>
    </>
  );
};

export default GifCard;
