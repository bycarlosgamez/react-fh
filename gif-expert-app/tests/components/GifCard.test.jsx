import { render } from '@testing-library/react';
import GifCard from '../../src/components/GifCard';

describe('<GifCard /> Tests', () => {
  const title = 'TEST';
  const url = 'https://google.com';
  test('must match with snapshot', () => {
    const { container } = render(<GifCard url={url} title={title} />);
    expect(container).toMatchSnapshot();
  });
});
