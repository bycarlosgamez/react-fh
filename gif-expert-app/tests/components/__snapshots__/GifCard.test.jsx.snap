// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifCard /> Tests must match with snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="TEST"
      src="https://google.com"
    />
    <p>
      TEST
    </p>
  </div>
</div>
`;
